travel-agency-api/src/main/java/cz/muni/fi/pa165/bluebat/dto/TripDTO.java

```diff
@@ -14,21 +17,22 @@
 @Setter
 public class TripDTO {
```

Comment:
Přemýšlím, jestli by nebylo dobré mít Ještě třeba TripDetailDTO, který by obsahoval i exkurze (Jejich nějakou zjednodušenou verzi třeba ExcursionListDTO, které by mělo třeba jen id a název), které by se mohli zobrazit na webu. TripDetailDTO by se vracelo třeba při getById 🤔

Reply:
To by malo ist cez reservation tam mas excursie.

Reply:
Ale ty chceš jako zákazník vidět, jaké tripy a exkurze jsou k dispozici, ne? Takže tripDetailDTO by mělo mít ExcursionDTO, abychom to mohli všechno zobrazit pohromadě. 

Napadlo mě totiž něco jako, že bude list tripů, když zákazníka nějaký zaujme, tak si ho může rozkliknout a vidět jednotlivé exkurze a ceny. No a pak by z toho detailu mohl kliknout na rezervaci, kde by si už jen vybral exkurze a vyplnil potřebné informace.

Reply:
Ked som na tym rozmyslal tak skor si myslim musime zobrazit Zoznam tripov bez excurzii. Potom ked si zakaznik vyberie trip zo excurziami, lebo na tej stranke bude aj pole na pridanie excurzii do Rezervacie. Takze ked kliknes na Trip vytvori sa rezervacia, tam si potom dovyplnis excurzie do rezervacie.  Taktiez to nebude vyzerat dobre ked mas napriklad 11 moznych excurzii v jednom tripe tak ti to zaberie polku strany.

Reply:
No jako hej, chceš udělat nějaký katalog tripů, který by zobrazoval jen seznam (tady bych asi využil jen nějaké zjednodušené schéma něco jako TripListDTO). No a po rozkliknutí by se mu zobrazil detail tripu, který by už obsahoval exkurze a jednotlivé ceny za položky. No a při tom rozkliknutí by se muselo zavolat API, které by podle id získalo TripDetailDTO, které by právě obsahovalo i ExcursionDTO

Reply:
Ok. Len rest vyuziva zoznamy TripListDTO bude List<TripSimplifiedDTO> to s konvertuje napriklad do JSON ako zoznam.

Reply:
Já vím, já spíš myslel TripListDTO jako ta jedna entita v tom seznamu, která nemusí obsahovat takové množství dat, která by se musela přesunou. Takže by obsahoval například jen název, destinaci a datum od/do 🤔